src/clj/ataru/applications/application_migration_store.clj

```diff
@@ -0,0 +1,54 @@
+(ns ataru.applications.application-migration-store
```

Comment:
Hmm, should we create a src/clj/ataru/db/migrations -folder and move this namespace there (maybe along with other earlier migration code if it exists?) This way this is a bit "in the way". Also could be named application_migration_1.24.clj?